src/openforms/contrib/kvk/validators.py

```diff
@@ -12,6 +14,8 @@
 
 from .client import NoServiceConfigured, SearchParams, get_kvk_search_client
 
+KVK_LOOKUP_CACHE_TIMEOUT = 60
```

Comment:
I think we can go for 5 minutes easily :)

Reply:
changed it to 5 mins